misc/phase_retrieval_tomopy_comparison.ipynb

```diff
@@ -0,0 +1,1264 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# -*- coding: utf-8 -*-\n",
+    "#  Copyright 2024 -  United Kingdom Research and Innovation\n",
+    "#  Copyright 2024 -  The University of Manchester\n",
+    "#\n",
+    "#  Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "#  you may not use this file except in compliance with the License.\n",
+    "#  You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "#  Unless required by applicable law or agreed to in writing, software\n",
+    "#  distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "#  See the License for the specific language governing permissions and\n",
+    "#  limitations under the License.\n",
+    "#\n",
+    "#   Authored by:    Hannah Robarts (STFC - UKRI)\n",
+    "#                   Laura Murgatroyd (STFC - UKRI)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Paganin Processor \n",
+    "This demo contains some examples and comparisons of using the `PaganinProcessor` phase retrieval methods in CIL "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook requires CIL v24.1.0 or greater, check the version below"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cil\n",
+    "print(cil.__version__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load some dependencies from CIL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cil.utilities import dataexample\n",
+    "from cil.processors import PaganinProcessor, TransmissionAbsorptionConverter, Padder, Normaliser\n",
+    "from cil.utilities.display import show2D\n",
+    "from cil.recon import FDK, FBP\n",
+    "from cil.io.utilities import HDF5_utilities\n",
+    "from cil.framework import AcquisitionGeometry, AcquisitionData"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook also requires TomoPy, numpy and matplotlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tomopy.prep.phase import retrieve_phase\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt \n",
+    "from scipy import constants"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Parallel beam data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we test the PaganinProcessor with parallel beam data. In the following cell\n",
+    "- Get a test parallel beam dataset \n",
```

Comment:
Explain that this is a test dataset from CIL and explain that you are taking a 2D slice etc

Reply:
Update now :)